bin/scons_dev_master.py

```diff
@@ -47,10 +47,8 @@
     #'tetex-bin',
     #'tetex-latex',
 
-    # for ubuntu 9.10 
-    # 'texlive-lang-french'
     'python3-sphinx',
-    'sphinx-rtd-theme-common',
+    'python3-sphinx-rtd-theme',
```

Comment:
We should probably just make a requirements.txt and/or add to setup.cfg where you can specify packages for testing/development, but not needed for running